include/mqt-core/dd/Package.hpp

```diff
@@ -1740,6 +1737,7 @@ template <class Config> class Package {
       return CachedEdge<Node>::zero();
     }
 
+    // TODO: this does also not yet work with identities
```

Comment:
Here this is a proper TODO as I am fairly sure that the current method does not handle identities well. Especially considering the variable index in the nodes.